Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -767,6 +795,12 @@ void ABenchmarkGymGameModeBase::SetStatTimer(const FString& TimeString)
 }
 #endif
 
+void ABenchmarkGymGameModeBase::SetMemReportIntervalTimer(const FString& TimeString)
+{
+	MemReportIntervalTimer.SetTimer(FCString::Atoi(*TimeString));
+	UE_LOG(LogBenchmarkGymGameModeBase, Log, TEXT("MemReport Interval is set to %s seconds"), *TimeString);
```

Comment:
This will add a log line every time SetMemReportIntervalTimer is called, and this get's called everytime the timer fires, it feels unnecessary. Recommend to add an int32 member for the interval and when processing the flag convert the string to int32 and log the value one time.

Reply:
Modified the SetMemReportIntervalTimer to InitMemReportIntervalTimer. It will be called before Tick() when the flag is set and save the interval to an int32 member.